packages/common/enabled-features/enabled-features.json

```diff
@@ -23,5 +26,7 @@
 
   "project_homepage:show_all_client_libraries": true,
   "project_homepage:show_instance_size": true,
-  "project_homepage:show_examples": true
+  "project_homepage:show_examples": true,
+
+  "table_editor:creation_enable_rls_toggle": true
```

Comment:
probably can generalize to enable_rls_toggle - cause this needs to cover the RLS button in the Table Editor header too